source/lex.tex

```diff
@@ -469,16 +470,25 @@
 
 \rSec1[lex.comment]{Comments}
 
-\pnum
 \indextext{comment|(}%
+\begin{bnf}
+\nontermdef{whitespace-character}\br
+    \unicode{0009}{character tabulation}\br
+    \textnormal{new-line}\br
+    \unicode{000b}{line tabulation}\br
+    \unicode{000c}{form feed}\br
+    \unicode{0020}{space}\br
+\end{bnf}
+
```

Comment:
Please move that to the very top of [lex.pptoken].  While the grammar is used here first, whitespace-characters are much more relevant for separating pptokens than for comments. Yes, we'll have a forward reference then. Hm. Not good, either.

Alternatively, a new subclause under [lex.char] would be appropriate: It's a kind of character (set), after all.
But having a section that contains just grammar is not good.  Can we add some more meat? Maybe not editorially.

Reply:
My first draft (not submitted) did indeed add the grammar to the top of [lex.pptoken], but it felt even more wrong there.

I think a real solution, leaning more in the direction of P2348 but remaining editorial, would be to add a new level above [lex.comment] for "Whitespace" the covers both whitespace characters, comprising the existing [lex.comment], and a new subclause with the grammar and some text describing whitespace characters.  This would be a sibling to [lex.char] and [lex.pptoken], similar to when we introduced [lex.char] for character sets.

Reply:
I have pushed a more substantial update that creates a new Whitespace clause for both whitespace tokens and comments.  With appropriate cross-references this feels like a better approach, and closer to P2348.  The main difference with that paper is that it defines a full grammar for whitespace, whereas my PR is limited to _whitespace-character_.  The full whitespace grammar in P2348 is more detailed, and has a special treatment for line-breaks that my PR deliberately leaves room for.

I believe my changes remain purely editorial with no semantic/technical changes, although Jens correctly points out that introducing a new grammar term should be seen in Core.  And while I would like to make this change for C++26, introducing the new Whitespace clause after our main clause re-org might make this C++29 material.

I have also retained my original branch that this replaces, should we wish to revert to the earlier approach.